Rename deployment command to stack

asalkeld opened the pull request:
1. remove "stack describe"
2. rename "stack new" to "new"
3. rename "deployment apply" to "stack up"
4. rename "deployment delete" to "stack down"
5. rename "deployment list" to "stack list"
6. remove user aliases
7. remove unimplemented command "provider list"
8. add aliases for up, list and down

fixes #137 #138

tjholm commented on packages/base/src/commands/run.ts:
```diff
@@ -499,8 +499,10 @@ export default class Run extends BaseCommand {
 			execa.sync('docker', ['ps']);
-		} catch {
-			throw new Error("Docker daemon not found! Ensure it's running.");
+		} catch (error) {
+			const origErrs: string[] = error.errors && error.errors.length ? error.errors : [error.stack];
+			cli.info("Docker daemon not found! Ensure it's running.");
+			throw new Error(`${origErrs.join('\n')}`);
```
There may be a more generic solution to this method using the base command error handling, to ensure it happens correctly for all error types.

jyecusch replied:
Why not just re-throw the original error?

raksiv commented:
Closing this pull request - new issue should be raised to handle consistently throwing errors.

asalkeld committed:
feat: rename commands to be more user friendly

1. remove "stack describe"
2. rename "stack new" to "new"
3. rename "deployment apply" to "stack up"
4. rename "deployment delete" to "stack down"
5. rename "deployment list" to "stack list"
6. remove user aliases

asalkeld committed:
chore: remove unimplemented command "provider list"

asalkeld committed:
feat: Add aliases for up,down and list
